artifacts/camara-cloudevents/subscription-notification-template.yaml

```diff
@@ -0,0 +1,759 @@
+openapi: 3.0.3
+info:
+  title: Notification Subscription Template
+  description: |
+    This file is a template for CAMARA explicit subscription template and for Notification model. Additional information are provided in API Design guidelines document.
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at CAMARA
+  url: hhttps://github.com/camaraproject/
+servers:
+  - url: '{apiRoot}/xxx-notification-subscriptions/v0'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+tags:
+  - name: xxx notification subscription
+    description: Operation to manage event subscription on xxx event 
+
+paths:
+  /subscriptions:
+    post:
+      tags:
+        - xxx notification subscription
+      summary: 'Create a xxx event subscription'
+      description: Create a xxx event subscription
+      operationId: createXxxSubscription
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - xxx:subscriptions:create
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SubscriptionRequest'
+        required: true
+      callbacks:
+        notifications:
+          "{$request.body#/webhook/notificationUrl}":
+            post:
+              tags:
+                - Session notifications callback
+              summary: "Session notifications callback"
```

Comment:
These callbacks are not only related to "Sessions". So I would propose to just call it `notifications callback`.
Wdyt?

Reply:
Agree !
Fixed